charts/piraeus/templates/operator-https-secret.yaml

```diff
@@ -0,0 +1,85 @@
+{{- $fullName := include "operator.fullname" . -}}
+{{- $cn := printf "%s-cs" $fullName -}}
+{{- $altName1 := printf "%s.%s" $cn .Release.Namespace }}
+{{- $altName2 := printf "%s.%s.svc" $cn .Release.Namespace }}
+{{- if (eq .Values.linstorHttpsMethod "helm") }}
+{{- $ca := genCA (printf "%s-ca" $fullName) 3650 -}}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "controller.httpsSecretName" . }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/resource-policy": "keep"
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+type: kubernetes.io/tls
+data:
+  {{- with genSignedCert $cn (list "127.0.0.1") (list $cn $altName1 $altName2 "localhost") 3650 $ca }}
```

Comment:
not tested, but should we also add `::1` as IP?

Reply:
added `::1`, is working fine:

```
X509v3 Subject Alternative Name:
    DNS:piraeus-op-cs, DNS:piraeus-op-cs.linstor, DNS:piraeus-op-cs.linstor.svc, DNS:localhost, IP Address:0:0:0:0:0:0:0:1, IP Address:127.0.0.1
```

```console
$ linstor --controllers 'https://[::1]:3371' c v
linstor controller 1.17.0; GIT-hash: 7e646d83dbbadf1ec066e1bc8b29ae018aff1f66
```